DigitalLearningSolutions.Web.Tests/Controllers/ForgotPassword/ForgotPasswordControllerTests.cs

```diff
@@ -22,24 +21,14 @@ public void SetUp()
             controller = new ForgotPasswordController(passwordResetService);
         }
 
-        [Test]
-        public void Render_basic_form_when_there_is_no_error_model()
```

Comment:
I might be missing something but why are we removing this test?

Reply:
I refactored the code around to use ModelState for errors rather than a custom error property (to keep things cleaner, and so that the generic _ErrorSummary partial view worked correctly). That test was making sure that when the custom error property was null, the basic form was shown. Without that property, that test would effectively become "the Index action returns the index page" which I did not think was really worth keeping in. Do we still want that (or something similar in its place)?

Reply:
Yes, I think it's worth keeping, especially with the redirect logic in place for logged in users.